src/test/java/racingcar/learningtest/SetCollectionTest.java

```diff
@@ -0,0 +1,53 @@
+package racingcar.learningtest;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+
+    @Test
+    @DisplayName("요구사항 1 > Set 사이즈 확인")
+    void getSizeOfSet() {
+        // given by setUp()
+
+        // when
+        int size = numbers.size();
+
+        // then
+        int expected = 3;
```

Comment:
아래 메서드로 컬렉션의 크기를 바로 검증할 수 있어요!

```suggestions
assertThat(numbers).hasSize(3);
```